graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -199,6 +203,24 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
   SharedMemoryPtr tensor_meta_shm_, tensor_data_shm_;
 };
 
+/**
+ * @brief Picks a specified number of neighbors for a node, starting from the
+ * given offset and having the specified number of neighbors.
+ *
+ * @param offset The starting edge ID for the connected neighbors of the sampled
+ * node.
+ * @param num_neighbors The number of neighbors to pick.
+ * @param fanout The number of edges to be sampled for each node. It should be
+ * >= 0 or -1. If -1 is given, all neighbors will be selected. Otherwise, it
+ * will pick the minimum number of neighbors between the fanout value and the
+ * total number of neighbors.
```

Comment:
I suppose this will be changed once `replace` option is added?

Reply:
Yes